### PR TITLE
Add Tcpdump.org.xml

### PR DESCRIPTION
**Pull request by J0WI**
#15750

### CODE REVIEW

**Comment by pipboy96** [comment]
More subdomains:
```
www.tcpdump.org
bpf.tcpdump.org
www.ca.tcpdump.org
www4.ca.tcpdump.org
lists.tcpdump.org
www.us.tcpdump.org
www4.tcpdump.org
```

File: src/chrome/content/rules/Tcpdump.org.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+	^ does not exist
+-->
+<ruleset name="Tcpdump.org">
+	<target host="www.tcpdump.org" />
+	<target host="www.ca.tcpdump.org" />
+	<target host="www.us.tcpdump.org" />
```
Comment (pipboy96) [inline src/chrome/content/rules/Tcpdump.org.xml]:
```
	<target host="www4.tcpdump.org" />
	<target host="www6.tcpdump.org" />
	<target host="www4.ca.tcpdump.org" />
	<target host="www6.ca.tcpdump.org" />
```

File: src/chrome/content/rules/Tcpdump.org.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!--
+	^ does not exist
```
Comment (pipboy96) [inline src/chrome/content/rules/Tcpdump.org.xml]:
```
	Mismatched:
		- bpf
		- www.ca1
		- www4.ca1
		- www6.ca1
		- lists
		- master
		- www6.us
```

Comment (J0WI) [reply src/chrome/content/rules/Tcpdump.org.xml]: done

**Pull request merged**